Fix aggregate payload mapping validation (#34)
astarte-platform/astarte-device-sdk-csharp: CHANGELOG.md

The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).

## [0.5.3] - Unreleased
### Fixed

> - Fix a bug on payload validations when sending object aggregated payload

Reviewer: ```suggestion
- Fix the payload validation on  the aggregated object interface.
```

Author: Changed.
